### PR TITLE
add channels group to rails profile

### PR DESCRIPTION
**Pull request by BenMorganIO**
This basically adds Channels as a group for test coverage. What I don't like about this is that it assumes (a) you're on Rails 5 and (b) that you're using ActionCable. I'm very open to modifying this PR to support Rails <5 and just do a detection for ActionCable.

### CODE REVIEW

**Comment by dtaniwaki** [comment]
👍

File: lib/simplecov/defaults.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@
   add_filter "/db/"
 
   add_group "Controllers", "app/controllers"
+  add_group "Channels", "app/channels"
```
Comment (bf4) [inline lib/simplecov/defaults.rb]: could be something like  `  add_group "Channels", "app/channels" if defined?(ActionCable) || defined?(Rails) && Rails::VERSION::MAJOR >= 5` but the below might be cleaner.   

```ruby
SimpleCov.profiles.define "rails5" do
  load_profile "rails"

  add_group "Channels", "app/channels"
end
```
 
@colszowka @xaviershay Any preference for adding the group here in all Rails version, with a condition, or making a new rails5 profile?

Comment (BenMorganIO) [inline lib/simplecov/defaults.rb]: Maybe just check for ActionCable. I know that in Rails 5 APIs, ActionCable isn't exactly used much :p

**Comment by PragTob** [comment]
Hi there,

thanks for the PR! I think as the others suggested just adding it when ActionCable is defined might be a good idea. 

If you have time, please go ahead and do it, otherwise I can also take over :)

**Comment by PragTob** [comment]
Merged as 5571d0d8e00744d :)

Thanks!

**Pull request closed without merging**